spiceaidocs/docs/components/data-connectors/s3.md

```diff
@@ -4,41 +4,22 @@ sidebar_label: 'S3 Data Connector'
 description: 'S3 Data Connector Documentation'
 ---
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
-
+Amazon S3 (Simple Storage Service) is a scalable object storage service that allows users to store, manage, and retrieve any amount of data from anywhere on the web.
```

Comment:
I don't think we need an intro for S3, given if you are looking at this page, and you have an S3 data source, you would likely know what it is.